purchases-sample/src/androidTest/java/IntegrationTest.kt

```diff
@@ -0,0 +1,43 @@
+
+import androidx.test.ext.junit.rules.activityScenarioRule
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.revenuecat.purchases.Purchases
+import com.revenuecat.purchases.getPurchaserInfoWith
+import com.revenuecat.sample.InitialActivity
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.fail
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import java.util.concurrent.CountDownLatch
+import java.util.concurrent.TimeUnit
+
+@RunWith(AndroidJUnit4::class)
+class IntegrationTest {
+
+    @get:Rule var activityScenarioRule = activityScenarioRule<InitialActivity>()
+
+    @Test
+    fun SDKCanBeConfigured() {
+        val scenario = activityScenarioRule.scenario
+        scenario.onActivity {
+            assertThat(Purchases.sharedInstance.appUserID).isNotNull()
+        }
+    }
+
+    @Test
+    fun purchaserInfoCanBeFetched() {
+        val lock = CountDownLatch(1)
+
+        val scenario = activityScenarioRule.scenario
+        scenario.onActivity {
+            Purchases.sharedInstance.getPurchaserInfoWith({
```

Comment:
here we're just testing that this can run correctly within the integration test process, right? if so, do we need Firebase Test Lab for it?

Reply:
Yes, this test will start the `InitialActivity`, which will initialize the SDK and then we get the purchaserInfo to see that everything worked fine. These tests are instrumented so they run in a real device.